### ISSUE TITLE
Not able to build the code

### ISSUE THREAD

**Issue by Ananyaaynana** [amaranand360/Cubic-Runout-Spline#1]
Hello. This is Ananya from Urbanmorph Consultants. 
I compiled Cubic_Runout_Spline.c like this `gcc -c Cubic_Runout_Spline.c -o spline.o`
This was successful. 
Then I linked the spline.o with test_code.c to be the executable of the test_code.c like this `gcc test_code.c spline.o -o test` but here I am getting this error:
![Screenshot from 2024-02-18 23-51-29](https://github.com/amaranand360/Cubic-Runout-Spline/assets/131510581/3af5c718-94b5-4754-ba98-e92dbb5900cf)
Is this the correct way to compile the program? If not please elaborate in the Readme document about the steps to compile and run this program.

**Comment by amaranand360** [comment]
@Ananyaaynana  yes correct ! use these command to compile the test  and fetch my latest changes there's a little mistake in test file

step 1 : `gcc -c cubic_runout_spline.c -o spline.o`

step 2 : `gcc -c test_code.c -o test_code.o`

step 3 : `gcc test_code.o spline.o -o test `
